inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -65,14 +66,20 @@ void GNADeviceHelper::setUpActiveList(const uint32_t requestConfigId, uint32_t l
     const auto status = Gna2RequestConfigEnableActiveList(requestConfigId, layerIndex, num_active_indices, ptr_active_indices);
     checkGna2Status(status);
 }
-void GNADeviceHelper::propagateSync(const uint32_t requestConfigId) {
-    wait(propagate(requestConfigId));
+void GNADeviceHelper::propagateSync(const uint32_t requestConfigId, Gna2AccelerationMode gna2AccelerationMode) {
+    wait(propagate(requestConfigId, gna2AccelerationMode));
 }
 
-uint32_t GNADeviceHelper::propagate(const uint32_t requestConfigId) {
+uint32_t GNADeviceHelper::propagate(const uint32_t requestConfigId, Gna2AccelerationMode gna2AccelerationMode) {
     uint32_t reqId;
-    const auto status = Gna2RequestEnqueue(requestConfigId, &reqId);
-    checkGna2Status(status);
+    if (gna2AccelerationMode == Gna2AccelerationModeHardware &&
+        detectedGnaDevVersion == Gna2DeviceVersionSoftwareEmulation) {
+        gnalog() << "GNA Device not detected, consider using other mode of acceleration";
```

Comment:
gnawarn() ?

Reply:
This was in the original code (in another place). But you are right. Will fix.

Reply:
Fixed